template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazDatalakeConfigProvider.java

```diff
@@ -25,7 +26,8 @@ public boolean isConfigurationNeeded(CmTemplateProcessor cmTemplateProcessor, Te
         return StackType.DATALAKE == source.getStackType()
                 && CMRepositoryVersionUtil.isRazConfigurationSupported(
                         source.getProductDetailsView().getCm().getVersion(), source.getCloudPlatform(), source.getStackType())
-                && source.getGeneralClusterConfigs().isEnableRangerRaz();
+                && source.getGeneralClusterConfigs().isEnableRangerRaz()
```

Comment:
Not needed ? Either SaaS DL or a PaaS DL in an environment. Don't think we're planning to deploy both at the same time. If we do - RAZ should be deployed on the DL.